ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainTestSubCommand.java

```diff
@@ -92,6 +94,41 @@ public class BlockchainTestSubCommand implements Runnable {
   // picocli does it magically
   @Parameters private final List<Path> blockchainTestFiles = new ArrayList<>();
 
+  /** Helper class to track test execution results for summary reporting. */
+  private static class TestResults {
+    private final AtomicInteger totalTests = new AtomicInteger(0);
+    private final AtomicInteger passedTests = new AtomicInteger(0);
+    private final AtomicInteger failedTests = new AtomicInteger(0);
+    private final Map<String, String> failures = new LinkedHashMap<>();
+
+    void recordPass(final String testName) {
+      totalTests.incrementAndGet();
+      passedTests.incrementAndGet();
+    }
+
+    void recordFailure(final String testName, final String reason) {
+      totalTests.incrementAndGet();
+      failedTests.incrementAndGet();
+      failures.put(testName, reason);
+    }
+
+    void printSummary(final java.io.PrintWriter out) {
+      out.println("\n" + "=".repeat(80));
```

Comment:
nit: probaby create constant for separator text such as:
```
private static final String SEP = "\n" + "=".repeat(80)";
```
Then in printSummary:
```
out.println(SEP);
```